### DISCUSSION TITLE
Default database schema

### DISCUSSION THREAD

**Discussion by meatandfat** [vendurehq/vendure#1993] [Q&A] [answered]
Hello.

**Where is the database schema created?** I’m looking for something like
`CREATE TABLE xxxxxxx (…);`

The closest I found was in https://github.com/vendure-ecommerce/one-click-deploy/blob/939d826caca7e86bc108934d2c0f138b5d50593f/src/populate.ts

I looked through https://socket.dev/npm/package/@vendure/core/files/1.9.1 and https://socket.dev/npm/package/@vendure/create/files/1.9.1/, but didn’t see anything that seemed to fit

I would like to compare the default to what I’ve designed on paper. Meaning, I already know I have to add tables. I'd like to see how the columns differ.

**Comment by michaelbromley** [comment, marked as answer]
Hi,
There is no file of SQL commands used to create the table structure. Rather, we use TypeORM to manage that by instead defining a set of entities which you can find [here](https://github.com/vendure-ecommerce/vendure/tree/master/packages/core/src/entity). These entity classes contain metadata which is used by TypeORM to generate the schema.

**Reply by meatandfat** [reply]
Thank you!
